src/lib.rs

```diff
@@ -1,35 +1,23 @@
-#![feature(asm, lang_items, naked_functions)]
```

Comment:
Does this mean the new libtock-rs doesn't need the nightly toolchain?

Reply:
Unfortunately not. Its dependencies need it, still. In `libtock-core` it should be in principle possible to remove the necessity for the nightly compiler (as in https://github.com/rust-embedded/cortex-m-rt), however I am not sure whether it is worth the effort, while the tock kernel needs its.

Reply:
I assume this is fixed?